### PR TITLE
skip-test: use `dev bazel generate` instead of `make bazel-generate`

### PR DESCRIPTION
**Pull request by otan**
Release note: None

### CODE REVIEW

**Review by rickystewart — APPROVED** [review]

File: pkg/cmd/skip-test/main.go [hunk 1]
```diff
@@ -145,7 +147,7 @@ func main() {
 	replaceFile(fileName, testName, issueNum)
 
 	// Update the package's BUILD.bazel.
-	if err := spawn("make", "bazel-generate"); err != nil {
+	if err := spawn("dev", "generate", "bazel"); err != nil {
```
Comment (rafiss) [inline pkg/cmd/skip-test/main.go:150]: is `./dev` safer?

Comment (otan) [reply pkg/cmd/skip-test/main.go:150]: hmm, probably.

**Review by otan — COMMENTED** [review]
bors r=rickystewart

now it checks both!

**Comment by otan** [comment]
bors r=rickystewart

**Pull request merged**